src/options/options.ts

```diff
@@ -117,7 +119,8 @@ function restore_options() {
       noHaptics:true,
       haply2diy:false,
       audio:true,
-       text:true
+      text:true,
+      language: ""
```

Comment:
Set default language of rendering to be auto, i.e., using user's browser UI Language directly if supported (English & French). Otherwise fallback to English.